package_test.go

```diff
@@ -575,12 +575,121 @@ func (s *PackageSuite) TestAllErrors(c *C) {
 			Commentf("\ntest %q failed:\ninput: %s\nslices: %s", t.summary, t.query, t.slices))
 	}
 
-	_, err = db.PlainDB().Exec(dropTables)
+	err = db.Query(nil, sqlair.MustPrepare(dropTables)).Run()
```

Comment:
This makes me think we'll probably want some sort of QueryString at some point, but let's wait until we have more data about this.

Reply:
I agree, it would be good to be to do a DML in one command, we could even accept inputs and pass the argument structs to `Prepare` behind the scenes.

Reply:
Indeed, that's probably uncontroversial enough for us to support right away.